starter-code/src/viking.js

```diff
@@ -1,11 +1,104 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+}
+Soldier.prototype.attack = function(){
+  return this.strength;
+}
+
+Soldier.prototype.receiveDamage = function(damage){
+
+ this.health -= damage;
+}
 
 // Viking
-function Viking() {}
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+Viking.prototype.receiveDamage = function(damage){
+
+  this.health -= damage;
+
+  if(this.health <= 0){
+      return this.name + " has died in act of combat";
+  } else { return this.name + " has received "+ damage +" points of damage"}
+}
+
+Viking.prototype.battleCry = function(){
+  return 'Odin Owns You All!'
+}
+
+function Viking(name, health, strength){
+  Soldier.call(this, health, strength);
+  this.name = name;
+}
 
 // Saxon
-function Saxon() {}
 
-// War
-function War() {}
+Saxon.prototype = Object.create(Soldier.prototype);
+Saxon.prototype.constructor = Saxon;
+Saxon.prototype.receiveDamage = function(damage){
+
+  this.health -= damage;
+
+  if(this.health <= 0){
+      return "A Saxon has died in combat";
+  } else { return  "A Saxon has received "+ damage +" points of damage"}
+}
```

Comment:
Haciendo la función `receiveDamage()` otra vez, estáis machacando la original en vez de reutilizar la que ya heredáis de `Soldier`. La manera correcta es reutilizar la función heredada modificándola para poder devolver los mensajes que os pide Jasmine; esto es así, llamar a la original `Soldier.prototype.receiveDamage.call(this, damage)` dentro de la nueva `Saxon.prototype.receiveDamage(damage)`. Del mismo modo es en la función anterior del mismo nombre de los vikingos